feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/elytron-tls-dependency/main/module.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+  ~ Copyright 2019 Red Hat, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.elytron-tls-dependency">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${org.wildfly.extras.elytron-tls:elytron-tls-dependency}"/>
+        <artifact name="${org.wildfly.security:wildfly-elytron-ssl-common}"/>
+    </resources>
+
+    <dependencies>
+        <!-- We need these dependencies to trigger weld/CDI -->
+        <module name="javax.enterprise.api"/>
+        <module name="org.jboss.weld.api"/>
+        <module name="org.jboss.weld.core"/>
+        <module name="org.jboss.weld.spi"/>
```

Comment:
I think this list needs to be updated. Take a look at what `wildfly-elytron-ssl-common` depends on.

Reply:
same as [last comment](https://github.com/wildfly-security-incubator/elytron-tls/pull/3#discussion_r1004917188)